app/.server/event-web-api/event-api.ts

```diff
@@ -21,25 +21,36 @@ export class EventApi {
 
     const search = new ProposalSearchBuilder(this.eventSlug, 'no-user', filters);
 
-    const proposals = await search.proposals({ reviews: false });
+    const proposals = await search.proposals();
 
     return {
       name: event.name,
+      startDate: event.conferenceStart,
+      endDate: event.conferenceEnd,
       proposals: proposals.map((proposal) => {
         return {
+          id: proposal.id,
           title: proposal.title,
           abstract: proposal.abstract,
+          deliberationStatus: proposal.deliberationStatus,
+          confirmationStatus: proposal.confirmationStatus,
+          publicationStatus: proposal.publicationStatus,
           level: proposal.level,
+          references: proposal.references,
           formats: proposal.formats.map((f) => f.name),
           categories: proposal.categories.map((c) => c.name),
           languages: proposal.languages as string[],
           speakers: proposal.speakers.map((speaker) => ({
             name: speaker.name,
             bio: speaker.bio,
             company: speaker.company,
+            references: speaker.references,
             picture: speaker.picture,
+            location: speaker.location,
+            email: speaker.email,
             socials: speaker.socials as SocialLinks,
           })),
+          reviews: proposal.reviews,
```

Comment:
What do you really need for reviews ? Here you get raw data of all individual reviews (maybe too much data).

You could use the `ReviewDetails` class to get the full summary of reviews.
- `average`: average review note of the proposal
- `positives`: number of positive reviews (heart)
- `negatives`: number of negative reviews (veto)

Instead of returning reviews raw data, you can return the global review by proposal:

```js
proposals: proposals.map((proposal) => {
  const reviews = new ReviewDetails(proposal.reviews);

  return {
    ...
    review: reviews.summary(), // review summary (note average, positives/negatives count)
    }
}
```

You could also add individual reviews, but I not sure it is really needed (it does not bring additional informations) :

```js
proposals: proposals.map((proposal) => {
  const reviews = new ReviewDetails(proposal.reviews);

  return {
    ...
    review: reviews.summary(), // review summary (note average, positives/negatives count)
    reviewDetails: reviews.ofMembers(), // list of individual reviews
}
```

Reply:
Having individual reviews would permit to analyse the behavior of the reviewers. That data is useful.

Reply:
I understand, but this analysis could be done directly in Conference Hall instead of exporting the data to process it in an other tool.

Which analysis of reviewers behaviour do you need? What't your use-cases?
We may be able to integrate it directly. 😉